go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/passageidentity/passage-go
 
-go 1.16
+go 1.22.0
```

Comment:
let's keep the go version at 16 for now and we'll bump it in another PR

Reply:
Will do!